src/resolvers.ts

```diff
@@ -84,7 +85,10 @@ const resolvers = {
       const validPassword = await bcrypt.compare(password, user.password);
 
       if (validPassword) {
-        const response = { user: user, token: "the_token" };
+        const token = jwt.sign({ username: user.email }, "supersecret", {
+          expiresIn: 3600,
+        });
+        const response = { user: user, token: token };
         return response;
```

Comment:
Quando vc tem uma variável com o mesmo nome do campo, vc pode omitir um deles:

```suggestion
        return { user, token };
```